tutorials/image/cifar10/cifar10.py

```diff
@@ -40,21 +40,25 @@
 import sys
 import tarfile
 
+import argparse
```

Comment:
Let's move this to the section above since this is a native Python import (right before `import os`)